BinaryObjectScanner/Protection/SecuROM.cs

```diff
@@ -25,7 +110,11 @@ public class SecuROM : IExecutableCheck<PortableExecutable>, IPathCheck
 
             if (exe.MatroschkaPackage != null)
             {
-                var matroschka = CheckMatroschkaPackage(exe, includeDebug);
+                var package = exe.MatroschkaPackage;
+                if (package == null)
+                    return null;
```

Comment:
This whole thing is redundant to the `if` block that it's in. If you want to get the package outside of the enclosing `if`, that would be far more appropriate.

Reply:
Not sure what happened to the logic here honestly, fixed.